pinc/TallyBoard.inc

```diff
@@ -164,7 +167,8 @@ class TallyBoard
             WHERE
                 tally_name      = '%s'
                 AND holder_type = '%s'
-                AND holder_id   = %s",
+                AND holder_id   = %s
```

Comment:
I realize this was already like that, but should this be `%d` instead of `%s`? (What caught my eye was that this `%s` is not quoted, and the others are.)

Reply:
It probably should, but I'd rather not muddy the water here by changing anything more than formatting if possible. It's functionally correct, just less precise, to render an int as a string.